packages/adapter-d1/src/d1.ts

```diff
@@ -161,10 +162,19 @@ class D1Transaction extends D1Queryable<StdClient> implements Transaction {
 }
 
 export class PrismaD1 extends D1Queryable<StdClient> implements DriverAdapter {
+  alreadyWarned = new Set()
+
   constructor(client: StdClient) {
     super(client)
   }
 
+  warnOnce = (key: string, message: string, ...args: unknown[]) => {
```

Comment:
Maybe add a comment for what's the current behavior? How often does it print?
[Once ....]